app/controllers/AuthenticationController.scala

```diff
@@ -66,6 +67,9 @@ class AuthenticationController @Inject()(
   private lazy val ssoKey =
     conf.WebKnossos.User.ssoKey
 
+  private lazy val isOIDCEnabled =
+    certificateValidationService.getFeatureOverrides.getOrElse("openIdConnectEnabled", true)
```

Comment:
Maybe this could use wkConf instead of the certificateValidationService directly? If I’ve seen it correctly there, the featureOverrides should automatically be applied there?